pkg/verifier/aws/entry_point.go

```diff
@@ -156,6 +156,11 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 
 	// Generate the userData file
 	// As expand replaces all ${var} (using empty string for unknown ones), adding the env variables used in userdata.yaml
+	if vei.Proxy.NoTls && vei.Proxy.Cacert != "" {
+		a.Logger.Info(vei.Ctx, " Cacert value will be ignored since NoTls is enabled.")
+		vei.Proxy.Cacert = ""
+	}
```

Comment:
```suggestion
	if vei.Proxy.NoTls {
		a.Logger.Info(vei.Ctx, "NoTls enabled; ignoring any provided CA certs")
		vei.Proxy.Cacert = ""
	}
```
If NoTls is specified, we don't care whether `vei.Proxy.Cacert` is blank or not; we're gonna ignore it. So why not simplify further?

Reply:
Done